app/test_main.py

```diff
@@ -1,3 +1,29 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_values() -> None:
+    assert get_human_age(0, 0) == [0, 0], "test zero values"
+
+
+def test_should_convert_into_zero_human_age() -> None:
+    assert get_human_age(14, 14) == [0, 0], "test convert to [0, 0] human ages"
+
+
+def test_should_convert_into_one_human_age() -> None:
+    assert get_human_age(15, 15) == [1, 1], "test convert to [1, 1] human ages"
+
+
+def test_should_convert_into_one_human_age2() -> None:
+    assert get_human_age(23, 23) == [1, 1], "test convert to [1, 1] human ages"
+
+
+def test_should_convert_into_two_human_ages() -> None:
+    assert get_human_age(24, 24) == [2, 2], "test convert to [2, 2] human ages"
+
+
+def test_should_convert_into_two_human_ages2() -> None:
+    assert get_human_age(27, 28) == [2, 2], "test convert to [2, 2] human ages"
+
+
```

Comment:
I don't see tests for all cases. Please check the checklist 
![image](https://user-images.githubusercontent.com/104218550/200385373-c0ade34d-720e-4a0c-bba9-24fb423fbbb2.png)